packages/babel-plugin-makepot/package.json

```diff
@@ -18,15 +18,22 @@
 	"bugs": {
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
+	"files": [
+		"build",
+		"build-module"
+	],
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"dependencies": {
 		"gettext-parser": "^1.3.1",
-		"lodash": "4.17.10"
+		"lodash": "^4.17.10"
```

Comment:
May have been mentioned already, but noting we're adding loose ranges for dependencies, which while not strictly within the scope of the effort is sensible given that these are libraries.

Reply:
Yes, I had to regenerate lock file because of the amount of changes Babel introduced after they started using namespaced packages so I thought we can also include this change for everything that we consider libraries for consistency. We were mixing ranges and exact versions before.